client/k8s/error.go

```diff
@@ -0,0 +1,30 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/juju/errgo"
+)
+
+var (
+	maskAny = errgo.MaskFunc(errgo.Any)
+)
+
+func maskAnyf(err error, f string, v ...interface{}) error {
```

Comment:
This is not necessary. We can simply use `microerror.MaskAny` and `microerror.MaskAnyf`. Here the import is then `microerror "github.com/giantswarm/microkit/error"`. Kubernetesd for instance does it like this.